virt_lightning/configuration.py

```diff
@@ -66,10 +69,7 @@ def __get(self, key):
 
     @property
     def libvirt_uri(self):
-        try:
-            return self.__get("libvirt_uri")
-        except:
-            pass
+        return self.__get("libvirt_uri")
```

Comment:
Code below will never been executed. Rollback or update.

Reply:
Will never be executed?

Reply:
Yes. You didn't delete check user:

![screen shot 2019-02-15 at 2 11 22 am](https://user-images.githubusercontent.com/296371/52823718-04e12580-30c7-11e9-8281-09e9df56a7bd.png)

Reply:
Aaah. It should be better with: https://github.com/virt-lightning/virt-lightning/commit/450b91c9556cef2e369f8f77f434b86134ef448e